about.html

```diff
@@ -0,0 +1,217 @@
+<!DOCTYPE html>
```

Comment:
- [ ] It is great to see you used your own design but please note you have to use the same design, color, layout, and fonts given by the university.